.github/pull_request_template.md

```diff
@@ -20,7 +20,7 @@ Link to any related issue(s):
 - [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
 - [ ] I have added any necessary documentation (if appropriate)
 - [ ] I have run make fmt and formatted my code
-- [ ] If changes include deprecations or removals, I defined an isolated PR with a relevant title as it will be used in the auto-generated changelog.
+- [ ] If changes include deprecations or removals I have added appropriate changelog entries.
```

Comment:
small pass by fix, this was outdated as we no longer need isolated PRs for deprecations or removals.